slack_bolt/listener/listener_error_handler.py

```diff
@@ -31,7 +31,7 @@ def handle(
 
 
 class CustomListenerErrorHandler(ListenerErrorHandler):
-    def __init__(self, logger: Logger, func: Callable[..., None]):
+    def __init__(self, logger: Logger, func: Callable[..., Optional[BoltResponse]]):
```

Comment:
Customizing a response to Slack had not been supported. Actually, there is no obstacle or strong reason for that.